src/qibojit/custom_operators/raw_kernels.py

```diff
@@ -500,4 +500,4 @@
 extern "C" __global__ void initial_state_kernel(T* state) {
   state[0] = T(1, 0);
 }
-"""
+"""  # pragma: no covers
```

Comment:
All the other use the imperative `cover`, so I guess the `s` is spurious:
```suggestion
"""  # pragma: no cover
```

https://coverage.readthedocs.io/en/stable/excluding.html